### ISSUE TITLE
Should we increase the lattice size before 1.0?

### ISSUE THREAD

**Issue by samreid** [phetsims/wave-interference#314]
The current lattice size was chosen to attain acceptable performance on iPad 2.  We settled on 101x101 with 40 cells in each dimension for padding (approximately the same dimension as in Java).  The odd number is so that there will be a single row and column for symmetry.

If I increase this to 201x201 with 40 cells in each dimension for padding, we have the following performance profile on iPad Air.

* 60 FPS on Waves Screen with Water source on
* 45 FPS on Waves Screen with Sound on, showing "both" particles and waves
* 60 FPS on Waves Screen with Light on, showing Graph and Screen.

The higher resolution lattice looks very smooth and clear:
![image](https://user-images.githubusercontent.com/679486/51139728-e7881580-1801-11e9-9c93-485169c8bbfb.png)

And this helps with the smoothness of the graphs as well:

![image](https://user-images.githubusercontent.com/679486/51139766-ff5f9980-1801-11e9-83a9-e36bc9871291.png)

We should discuss this before publishing 1.0 because: even thought it will be possible to change the lattice size in a future update, it may be difficult to do so while exactly preserving the physics.  I don't want to get into a situation where we publish 1.0 with a small lattice, then want to increase the lattice for a newer version and either (a) end up disrupting a lot of the physics for teachers that have written quantitative activities or (b) force us to waste a lot of extra time trying to get the quantitative behavior to match the 1.0 release but with a larger lattice or (c) decide not to improve the simulation for fear of breaking compatibility with existing lessons.

I also tried a lattice size of 301x301 (with same damping regions) and it degraded to 20FPS on iPad Air after 15-20 seconds.  I'm not saying 201x201 is perfect--if we decide to increase before 1.0 we should start by fine tuning that number--maybe even a bit higher is better.

**Comment by ariel-phet** [comment]
Discussed over Slack...summary is @arouinfar will test on chromebook with 201x201 and if all looks reasonable, @samreid will proceed with re-calibration of the model and moving towards a second dev test.

**Comment by arouinfar** [comment]
I spent an hour testing dev.62 with on Chadwick with a lattice size of 201. During testing, I would have lots of options on -- Graph, Screen, Intensity, Particles, Wave Detector. I would also rapidly scrub sliders to create noise in the lattice.  The Waves and Slits screens consistently remained 50+ FPS, and the frame rate was often close to 60. On Interference, the performance was generally similar, but would occasionally drop into 20 FPS territory. However, the sim never felt slow or jumpy. This low frame rate would sporadically show up, and I think it would occur when rapidly changing source separation (hard to tell, since it was inconsistent).

I also tested dev.62 with a lattice size of 301. I didn't really notice much of a boost in graphic quality, but it did knock off ~10 FPS on each screen. I also observed the inconsistent frame rate drop on the Interference screen.

@samreid @ariel-phet I think a lattice size of 201 works quite well on the Chromebook. I'm not quite sure what to make of the occasional drop in frame rate on Interference. Since it happens sporadically, and doesn't really feel or look all that jumpy, I'd say it's acceptable.
